### PR TITLE
Rework some briefcases, add some to the loadout

### PR DESCRIPTION
**Pull request by TheGoret**
Happy_Fox made sprites for briefcases and I decided to add them. He also reworked some basic hand-sprites, such as the weird outline on the brown one. 

Thanks to Happy_Fox for the spriting, and Mwahahahaha for the code from a closed PR.

### CODE REVIEW

**Comment by TheGoret** [comment]
!review

File: code/game/objects/items/weapons/storage/briefcase.dm [hunk 1]
```diff
@@ -11,4 +11,22 @@
...
 	use_sound = 'sound/items/storage/briefcase.ogg'
-	drop_sound = 'sound/items/drop/backpack.ogg'
\ No newline at end of file
+	drop_sound = 'sound/items/drop/backpack.ogg'
+
+/obj/item/storage/briefcase/black
+	name = "black briefcase"
+	icon_state = "briefcase_black"
+	item_state = "briefcase_black"
+	desc ="Named for lawyers carrying briefs in a case. It's about time they had something named for them, isn't it?"
```
Comment (Geevies) [inline code/game/objects/items/weapons/storage/briefcase.dm]:
```suggestion
	desc = "Named for lawyers carrying briefs in a case. It's about time they had something named for them, isn't it?"
```

File: code/game/objects/items/weapons/storage/briefcase.dm [hunk 2]
```diff
@@ -11,4 +11,22 @@
...
+
+/obj/item/storage/briefcase/aluminium
+	name = "metal briefcase"
+	desc = "A heavy-duty briefcase for your most important documents."
+	icon_state = "briefcase_alum"
+	item_state = "briefcase_alum"
+
+/obj/item/storage/briefcase/nt
+	name = "nt briefcase"
```
Comment (Geevies) [inline code/game/objects/items/weapons/storage/briefcase.dm]:
```suggestion
	name = "\improper NT briefcase"
```

**Comment by NonQueueingMatt** [comment]
Icon conflicts.

**Comment by TheGoret** [comment]
its ugly and messy so I'mma make it on a fresh branch

**Pull request closed without merging**